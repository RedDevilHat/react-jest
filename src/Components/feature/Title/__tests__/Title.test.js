import React from "react";

describe("Render Title component", () => {
    testRenderTitle();
});
