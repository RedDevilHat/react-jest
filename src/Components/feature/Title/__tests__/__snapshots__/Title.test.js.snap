// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Title component render UI component 1`] = `
<div
  className="title"
>
  <h2>
    MY FIRST TESTING PROJECT
  </h2>
</div>
`;
